internal/olm/operator/registry/index_image.go

```diff
@@ -50,39 +50,23 @@ func (c IndexImageCatalogCreator) CreateCatalog(ctx context.Context, name string
 		return nil, fmt.Errorf("get database path: %v", err)
 	}
 
-	fmt.Printf("IndexImageCatalogCreator.IndexImage:        %q\n", c.IndexImage)
```

Comment:
Are these logs being removed for a good reason? I thought we wanted to provide as much information as possible to the user.

Reply:
Ditto https://github.com/operator-framework/operator-sdk/pull/3832#discussion_r483631929